lib/commands.js

```diff
@@ -221,8 +221,12 @@ module.exports = (message, command, args) => {
         headers: {
           'Content-Type': 'application/json'
```

Comment:
Add ``'Accept': 'application/json'`` so the response is json

Reply:
Updated